assets/src/edit-story/app/story/useStoryReducer/test/combineElements.js

```diff
@@ -228,6 +228,97 @@ describe('combineElements', () => {
       height: 1,
     });
   });
+
+  describe('combine elements with links', () => {
+    it('should not preserve link if combining with background element', () => {
+      const { restore, combineElements } = setupReducer();
+
+      restore(getDefaultState4());
+
+      // Combine element 456 into 123
+      const result = combineElements({ firstId: '456', secondId: '123' });
+
+      expect(result.pages[0].elements[0]).toStrictEqual({
+        id: '123',
+        isBackground: true,
+        flip: {},
+        focalX: 50,
+        focalY: 50,
+        height: 10,
+        resource: {
+          src: '1',
+          type: 'image',
+        },
+        scale: 100,
+        type: 'image',
+        width: 10,
+        x: 10,
+        y: 10,
+      });
+    });
+
+    it('should preserve the origin element link if combining with another', () => {
+      const { restore, combineElements } = setupReducer();
+
+      restore(getDefaultState4());
+
+      // Combine element 456 into 789
+      const result = combineElements({ firstId: '456', secondId: '789' });
+
+      expect(result.pages[0].elements[1]).toStrictEqual({
+        flip: {},
+        focalX: 50,
+        focalY: 50,
+        height: 10,
+        id: '789',
+        link: {
+          url: 'https://link456.com/',
+          icon: 'https://link456.com/image.png',
+          desc: 'Lorem ipsum dolor',
+        },
+        resource: {
+          src: '1',
+          type: 'image',
+        },
+        scale: 100,
+        type: 'image',
+        width: 10,
+        x: 10,
+        y: 10,
+      });
+    });
+
+    it('should preserve the target element link if the origin is without link', () => {
+      const { restore, combineElements } = setupReducer();
+
+      restore(getDefaultState4());
+
+      // Combine element 456 into 789
+      const result = combineElements({ firstId: '007', secondId: '789' });
+
+      expect(result.pages[0].elements[2]).toStrictEqual({
+        height: 10,
+        id: '789',
+        link: {
+          url: 'https://link789.com/',
```

Comment:
That domain actually exists 😲 !

[gTLD `.example`](https://en.wikipedia.org/wiki/.example) is recommended for this kind of usage.